doc/cookbook/source/examples/classifier/relaxedtree.rst

```diff
@@ -0,0 +1,41 @@
+=======================
```

Comment:
Please remove Classifier in the title (It is listed under classifier)
Rename to relaxed_tree.tst